fdbclient/BackupContainerFileSystem.actor.cpp

```diff
@@ -613,13 +688,25 @@ class BackupContainerFileSystemImpl {
 					s.restorable = false;
 				if (!desc.contiguousLogEnd.present() || desc.contiguousLogEnd.get() <= s.endVersion)
 					s.restorable = false;
+				// If there is logs gap after contiguousLogEnd, then check whether the current snapshot
+				// can be restored from the logs available after contiguousLogEnd.
+				if (desc.contiguousLogEnd.present() && desc.contiguousLogEnd.get() <= s.beginVersion) {
```

Comment:
Do we want to check when `desc.contiguousLogEnd.get() == s.beginVersion`?

Reply:
Agree. Changed the code to `desc.contiguousLogEnd.get() < s.beginVersion`